stacks-signer/src/runloop.rs

```diff
@@ -72,6 +77,16 @@ pub enum State {
     Sign,
 }
 
+/// Additional Info about a proposed block
+pub struct BlockInfo {
+    /// The block we are considering
+    block: NakamotoBlock,
+    /// Our vote on the block if we have one yet
+    vote: Option<Vec<u8>>,
```

Comment:
What's the `Vec<u8>` here?

Reply:
Thjis is the signature hash or the signature hash + "n" to indicate a no vote. We must store this to the side as a cooridnator could be malicious and overwrite with a fake message in the signature request OR because we lost our vote but want to remain consistent in our voting. Also to enable a signer nonce request to be over an entire serialized block while the response be over its hash (or of course the hash + b'n' to indicate rejection)